cyral/resource_cyral_repository_identity_map.go

```diff
@@ -256,14 +256,17 @@ func resourceRepositoryIdentityMap(deprecate bool) *schema.Resource {
 			"repository_id": {
 				Type:     schema.TypeString,
 				Required: true,
+				ForceNew: true,
 			},
 			"repository_local_account_id": {
 				Type:     schema.TypeString,
 				Required: true,
+				ForceNew: true,
 			},
 			"identity_type": {
 				Type:     schema.TypeString,
 				Required: true,
+				ForceNew: true,
 			},
 			"identity_name": {
```

Comment:
I tried updating the `identity_name`, but this created a new resource without deleting the old one, so this argument should include the `ForceNew` too. For some reason, this is only not necessary for the access_duration. Other than that, everything else looks great, just add this change and I'll approve it.